pkg/app/pagewriter/pagewriter.go

```diff
@@ -47,7 +47,10 @@ type Opts struct {
 	DisplayLoginForm bool
 
 	// ProviderName is the name of the provider that should be displayed on the login button.
-	ProviderName string
+	ProviderName []string
+
+	// ProviderID is the unique ID of each provider that we use to create the start path for each provider.
+	ProviderID []string
```

Comment:
Seems like these need to be paired together no? To avoid a mismatch of ID and name, should these be a map?